apps/communications/contacts/elements/form.html

```diff
@@ -135,7 +135,7 @@ <h1 id='contact-form-title' data-l10n-id="addContact"></h1>
                 <button type="reset">Clear</button>
               </p>
             </div>
-            <div class="with-overflow">
+            <div class="with-overflow companyNameHidden">
```

Comment:
bad idea to use class name instead of Id (according to your usage of it). ClassName could be easily assigned to any number of elements, ``document.querySelector('classNam')`` will return **only first** occurrence of such item. If you want to have 'short-cut' to particular item in DOM - use ``id``.